### PR TITLE
feat: added links to documentation

### PR DESCRIPTION
**Pull request by Toxer10**
## Changes

- added links to documentation

### CODE REVIEW

File: src/components/organisms/ActionsPane/ActionsPane.tsx [hunk 1]
```diff
@@ -306,13 +307,13 @@ const ActionsPane = (props: {contentHeight: string}) => {
               onChange={k => setKey(k)}
               tabBarExtraContent={
                 selectedResource && resourceKindDocumentation?.helpLink ? (
-                  <Button
+                  <StyledExtraRightButton
```
Comment (olensmar) [inline src/components/organisms/ActionsPane/ActionsPane.tsx:310]: we should add a tooltip - "Open Kubernetes documentation for this resource type in external browser"?

Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx:310]: Yes!

Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx]: Done

File: src/components/organisms/ActionsPane/ActionsPane.tsx [hunk 2]
```diff
@@ -306,13 +307,15 @@ const ActionsPane = (props: {contentHeight: string}) => {
...
               tabBarExtraContent={
                 selectedResource && resourceKindDocumentation?.helpLink ? (
-                  <Button
-                    onClick={() => openExternalResourceKindDocumentation(resourceKindDocumentation?.helpLink)}
-                    type="link"
-                    ghost
-                  >
-                    See {selectedResource?.kind} documentation <BookOutlined />
-                  </Button>
+                  <Tooltip title="Open Kubernetes documentation for this resource type in external browser">
```
Comment (olensmar) [inline src/components/organisms/ActionsPane/ActionsPane.tsx:310]: move to tooltips.ts please...

Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx]: Done

**Review by olensmar — APPROVED** [review]

**Pull request merged**